### DISCUSSION TITLE
How to use scss with @use instead of @import which will be deprecated

### DISCUSSION THREAD

**Discussion by rallets** [twbs/bootstrap#39591] [Q&A] [answered]
Hi, I'm wondering how is it possible to "import" the bootstrap `scss` files for customization using `@use` instead of `@import` as stated in the docs:
[https://getbootstrap.com/docs/5.3/customize/sass/](https://getbootstrap.com/docs/5.3/customize/sass/)
In my understanding `@import` should be deprecated in the future.
Is it possible as today, or it requires a lot of code changes in bootstrap?
Thanks

**Comment by coliff** [comment, marked as answer]
I think it's planned for v6:
https://github.com/twbs/bootstrap/issues/29853#issuecomment-1130207460

**Reply by mdo** [reply]
That's right!
